dist/setup-appliance.sh

```diff
@@ -654,6 +688,9 @@ export LC_ALL=C
 prepare_os_settings
 
 ENABLE_OPTIONAL_SERVICES=0
+ENABLE_FORCEPROJECTKEYS=1
```

Comment:
I think this needs discussion. I'm kind of opposed to have forceprojectkeys enabled as a default.

Reply:
No strong opinions here - I leave it up to you and @adrianschroeter to decide the default behaviour.
This has been a "regression" (which is in the code now for 7 years), so I tried to implement the behaviour like before.

And if you would ask me if is smart to "fix" and 7 year old regression - my clear answer - NO

So if you and @adrianschroeter would agree, I would change that.